### PR TITLE
Migrate to SymfonyConnect

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,12 +20,12 @@
...
     "autoload": {
         "psr-0": {
-            "SensioLabs": ["src/"]
+            "SymfonyCorp": ["src/"]
         }
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "4.x-dev"
+            "dev-migrate-to-symfonyconnect": "5.x-dev"
```
Comment (tucksaun) [inline composer.json]:
```suggestion
            "dev-master": "5.x-dev"
```

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 SensioLabs
+Copyright (c) 2018 Symfony
```
Comment (lyrixx) [inline LICENSE]: 2019?

**Comment by nicolas-grekas** [comment]
Shouldn't archive the current repository, and create a new one for the new package?
I fear packagist/composer won't like having two different package in the same repo.
I'd suggest naming the new package `symfonycorp/connect-sdk`, WDYT?

**Comment by nicolas-grekas** [comment]
Merged as part of #76

**Pull request closed without merging**